packages/contracts/contracts/GenArt721CoreV3.sol

```diff
@@ -1176,6 +1176,10 @@ contract GenArt721CoreV3 is
                 }
             }
             revert("Improperly formatted aspect ratio");
+
+            unchecked {
```

Comment:
This introduces a bug because it is unreachable code - we use continue statements above.
I think sacrificing some gas efficiency for iterator incrementing readability is probably ideal here, so prefer the original pattern of incrementing the iterator in line 1161.

Reply:
hmm, let me check it out

Reply:
fixed in [2a0930d](https://github.com/ArtBlocks/artblocks-contracts/pull/1212/commits/2a0930d3a8fe460a81b39e74681d7623b2ac0498)